monkey/infection_monkey/utils.py

```diff
@@ -35,3 +36,25 @@ def utf_to_ascii(string):
     # Converts utf string to ascii. Safe to use even if string is already ascii.
     udata = string.decode("utf-8")
     return udata.encode("ascii", "ignore")
+
+
+def create_monkey_dir():
```

Comment:
Feels like we can/should add a meta configuration variable to the monkey that fixes all these if statements.

Reply:
Add a function get_monkey_dir path that does the if statement. You can use memorization for it